ert3_examples/spe1/stages.yml

```diff
@@ -14,30 +14,17 @@
 
   output:
     -
-      record: wopt_prod
-      location: WOPT:PROD.json
-    -
-      record: wwpt_prod
-      location: WWPT:PROD.json
-    -
-      record: wgpt_prod
-      location: WGPT:PROD.json
-    -
-      record: wwit_inj
-      location: WWIT:INJ.json
-
-
+      record: eclsum
+      location: case/SPE1
+      keys: [WOPT:PROD, WWPT:PROD, WGPT:PROD, WWIT:INJ]
+      # check where these ends up, in sqlite??
```

Comment:
You can at least run `ert3 export evaluation` to export the data after evaluation and inspect it.